### DISCUSSION TITLE
Setup

### DISCUSSION THREAD

**Discussion by raf1111** [Koenkk/zigbee2mqtt#5686] [General]
Is there somewhere a tutorial about how to setup zigbee2mqtt with the TI Launchpad CC26X2R1 

I installed the firmware already on the CC26X2R1 
I installed the Mosquitto broker on Homeassistant 
I installed the Zigbee2mqtt and Zigbee2mqtt Edge on Homeassistant 
I added the correct port >-
    /dev/serial/by-id/usb-Texas_Instruments_XDS110__03.00.00.13__Embed_with_CMSIS-DAP_L1100LFR-if00
    to Zigbee2mqtt in the configuration. 
    
I try to start the application but in the logs it show me some error.

Zigbee2MQTT:info  2021-01-10 23:45:08: Logging to console and directory: '/share/zigbee2mqtt/log/2021-01-10.23-45-07' filename: log.txt
Zigbee2MQTT:info  2021-01-10 23:45:08: Starting Zigbee2MQTT version 1.17.0 (commit #1.17.0)
Zigbee2MQTT:info  2021-01-10 23:45:08: Starting zigbee-herdsman (0.13.46)
Zigbee2MQTT:error 2021-01-10 23:45:27: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2021-01-10 23:45:27: Failed to start zigbee
Zigbee2MQTT:error 2021-01-10 23:45:27: Exiting...
Zigbee2MQTT:error 2021-01-10 23:45:27: Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)
    at ZStackAdapter.<anonymous> (/zigbee2mqtt-1.17.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/adapter/zStackAdapter.js:94:31)
    at Generator.throw (<anonymous>)
    at rejected (/zigbee2mqtt-1.17.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/adapter/zStackAdapter.js:25:65)
npm
 ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR!
 zigbee2mqtt@1.17.0 start: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zigbee2mqtt@1.17.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-01-10T22_45_27_835Z-debug.log
2021-01-10T23:45:28: PM2 log: App [npm:0] exited with code [1] via signal [SIGINT]
2021-01-10T23:45:28: PM2 log: App [npm:0] starting in -fork mode-
2021-01-10T23:45:28: PM2 log: App [npm:0] online
> zigbee2mqtt@1.17.0 start /zigbee2mqtt-1.17.0

Did I miss something in the setup ?

**Comment by TheNutHouse84** [comment]
I don't know if you saw this page:

https://www.zigbee2mqtt.io/information/supported_adapters.html#texas-instruments-launchxl-cc26x2r1

Specifically:
When migrating from another dongle (e.g. CC2531), make sure to modify your pan_id in your configuration, otherwise Zigbee2MQTT won’t start.

**Comment by raf1111** [comment]
The button close to the connector worked to start the device

**Comment by raf1111** [comment]
Thanks for the Help 

I have an other question. 
I already set up my 64 devices correctly and also created some automations in Home assistant and everything worked perfect.
I needed to restart my server to add some RAM.

Now it seems like the Phillips Lamps (repeater) are disconnected.
Some devices are still here but that's endpoints. 

Zigbee2MQTT:info  2021-01-22 20:37:08: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'state' to 'Licht Terrasse / Licht Auffahrt hinten' failed: 'Error: Command 0x847127fffe214d15/2 genOnOff.on({}, {\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":false,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null}) failed (Data request failed with error: 'MAC channel access failure' (225))'","meta":{"friendly_name":"Licht Terrasse / Licht Auffahrt hinten"},"type":"zigbee_publish_error"}'
Zigbee2MQTT:error 2021-01-22 20:37:16: Publish 'set' 'state' to 'Licht Terrasse / Licht Auffahrt hinten' failed: 'Error: Command 0x847127fffe214d15/1 genOnOff.on({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) failed (Data request failed with error: 'MAC channel access failure' (225))'
Zigbee2MQTT:info  2021-01-22 20:37:16: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'state' to 'Licht Terrasse / Licht Auffahrt hinten' failed: 'Error: Command 0x847127fffe214d15/1 genOnOff.on({}, {\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":false,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null}) failed (Data request failed with error: 'MAC channel access failure' (225))'","meta":{"friendly_name":"Licht Terrasse / Licht Auffahrt hinten"},"type":"zigbee_publish_error"}'

**Reply by Koenkk** [reply]
225 = interference, probably from wifi: https://www.metageek.com/training/resources/zigbee-wifi-coexistence.html

**Reply by raf1111** [reply]
But I don't understand why the the half of my devices work. 
For example the end nodes work. 
There have a connection 

Only the repeaters which are connected to 230V are not reachable

**Reply by raf1111** [reply]
Wifi is in Channel:11
Zigbee: 15

**Reply by raf1111** [reply]
Changed now the W-lan to 1 and Zigbee to 24. Lets see if this works better

**Comment by raf1111** [comment]
Is there a solution for this Problem 

I made a update for Homeassistant and now everything is gone again.
